plugins/offboard/offboard.h

```diff
@@ -152,6 +158,13 @@ class Offboard : public PluginBase
      */
     void set_velocity_body(VelocityBodyYawspeed velocity_body_yawspeed);
 
+    /**
+     * @brief Set the actuator control targets for a specific control group.
+     *
+     * @param actuator_control_target float[8] and control_group uint8_t.
```

Comment:
Docs is subset of doxygen standard so you will need the following (doxgen will pull in types). 
```
@param actuator_control_target Set the actuator control target.
@ control_group uint8_t Set the actuator control group.
```
```